function-images/tests/chained-function-eventing/README.md

```diff
@@ -0,0 +1,59 @@
+# Knative Eventing μBenchmarks
+
+## Design
+We present a very simple _hello world_ pipeline where
+- **producer** accepts incoming `Greeter` gRPC requests from outside world;
+- **consumer** digests `greeting` events produced by the _producer_.
+
+Both services log extensively for debugging and/or testing.
+
+### Benchmarking Details
+To be able to time the pipeline, the **producer** registers another gRPC server called _signaller_. Upon receiving a request, the **producer** creates a CloudEvent with `traceid` extension (i.e. header) set to a random UUID, and waits for a _signal_ from the consumer. The **consumer**, once done, calls `Signaller.Signal(Completion{ traceId: XXX })` over gRPC on the **producer** to signal the completion of the eventing pipeline. Upon the receipt of such signal, the **producer** returns a gRPC response to the client, which in turn can measure the end-to-end latency of an eventing invocation.
```

Comment:
please update it later on

Reply:
You mean with respect to the timeseries DB?